plugins/scaffolder-backend/src/lib/index.ts

```diff
@@ -14,5 +14,4 @@
  * limitations under the License.
  */
 
-export * from './catalog';
```

Comment:
Should also remove `CatalogEntityClient.ts` itself right? Can't see that being removed here

Reply:
Oh derp, it must have saved it again when I was looking at the old version. Coming right up!